app/helpers/application_helper/toolbar/middleware_server_center.rb

```diff
@@ -67,15 +67,21 @@ class ApplicationHelper::Toolbar::MiddlewareServerCenter < ApplicationHelper::To
           nil,
           N_('Suspend this server'),
           N_('Suspend Server'),
-          :image   => "suspend",
-          :confirm => N_("Do you want to suspend this server?")),
+          :image => "suspend",
+          :data  => {'toggle'        => 'modal',
+                     'target'        => '#modal_param_div',
+                     'function'      => 'miqCallAngular',
+                     'function-data' => '{"name": "showListener", "args": ["operation:suspend"]}'}),
         button(
           :middleware_server_resume,
           nil,
           N_('Resume this server'),
           N_('Resume Server'),
           :image   => "resume",
-          :confirm => N_("Do you want to resume this server?")),
+          :data  => {'toggle'        => 'modal',
+                     'target'        => '#modal_param_div',
+                     'function'      => 'miqCallAngular',
+                     'function-data' => '{"name": "showListener", "args": ["operation:resume"]}'}),
```

Comment:
I think resume has no timeout. 

See e.g. https://github.com/pilhuhn/hawkular-agent/blob/c20716810a14552dadc1ebee7747ca7dec634d2d/hawkular-wildfly-agent-feature-pack/src/main/resources/subsystem-templates/hawkular-wildfly-agent.xml#L612 for a list of operations and parameters (we do not need to support the 'blocking' parameter).